src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -275,6 +275,9 @@ void DistributedSearchTreeImpl<DeviceType>::deviseStrategy(
     ExecutionSpace const &space, Predicates const &queries,
     DistributedTree const &tree, Indices &indices, Offset &offset, Distances &)
 {
+  Kokkos::Profiling::pushRegion(
```

Comment:
I see `pushRegion` but no corresponding `popRegion` here and several other functions.